vpc-subnet-test-prerequisites/main.tf

```diff
@@ -0,0 +1,70 @@
+terraform {
+  required_version = "~> 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+    template = {
+      source  = "hashicorp/template"
+      version = "~> 2.2.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.aws_region
+  alias  = "target"
```

Comment:
"target" not used alias. I think clearer if removed - unless there is a reason to keep it. Or use it.

Reply:
apparently my tests fail when i remove the alias